### DISCUSSION TITLE
Azure App Service Deployment - Custom Recognizers?

### DISCUSSION THREAD

**Discussion by TalkTomeG00se** [microsoft/presidio#1223] [Q&A]
I've successfully deployed both the analyzer and anonymizer into Azure App Service using the Presidio supplied ARM template [here](https://microsoft.github.io/presidio/samples/deployments/app-service/) and everything is working well. I've been looking around the documentation and couldn't find a solid answer to my question. Is it possible to add custom recognizers once the Presidio containers have been deployed into the Azure App Service? I have some custom regex patterns I want to add, deny lists etc.

**Comment by omri374** [comment]
Hi,
This requires you to change the code in app.py:
https://github.com/microsoft/presidio/blob/883afe7a5d49f9bf6d3717374e1688fb63b3e48e/presidio-analyzer/app.py#L40

You should replace the call to the vanilla `AnalyzerEngine` with your own custom one, which could have new types of recognizers, nlp models, context aware enhancers etc.
